apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/index.js

```diff
@@ -11,20 +10,18 @@ import { __ } from '@wordpress/i18n';
 import edit from './edit';
 import './style.scss';
 
-if ( 'wp_template' === fullSiteEditing.editorPostType ) {
```

Comment:
Maybe we could invert this check to `if ( 'wp_template_part' !== fullSiteEditing.editorPostType)`. The rationale being that we don't want to allow arbitrary nesting of template parts and potential circular dependencies leading to infinite loops. In previous case this was resolved by allow them only to be inserted on `wp_template` CPT, but making an exception for `wp_template_part` could work too.

Reply:
Fixed on c14efe2696789736736c8065eccf4c044f5275a1.

One thing that we'll have to fix, is that _Template Parts_ can be added to the _Post Content_ now.

![FireShot Capture 013 - Edit Page ‹ Gutenberg Dev — WordPress - localhost](https://user-images.githubusercontent.com/233601/59463943-876fbc80-8dfd-11e9-81c1-f7185b0b66a9.png)